src/crypto/public_key/elliptic/indutnyKey.js

```diff
@@ -0,0 +1,84 @@
+// OpenPGP.js - An OpenPGP implementation in javascript
+// Copyright (C) 2015-2016 Decentral
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 3.0 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+/**
+ * @fileoverview Wrapper for a KeyPair of an curve from indutny/elliptic library
+ * @requires enums
+ * @requires asn1.js
+ * @module crypto/public_key/elliptic/indutnyKey
+ */
+
+import { loadScript, dl } from '../../../lightweight_helper';
+import config from '../../../config';
+import util from '../../../util';
+
+export function keyFromPrivate(indutnyCurve, priv) {
+  const keyPair = indutnyCurve.keyPair({ priv: priv });
+  return keyPair;
+}
+
+export function keyFromPublic(indutnyCurve, pub) {
+  const keyPair = indutnyCurve.keyPair({ pub: pub });
+  if (keyPair.validate().result !== true) {
+    throw new Error('Invalid elliptic public key');
+  }
+  return keyPair;
+}
+
+let ellipticPromise;
+
+/**
+ * Load elliptic on demand to the window.openpgp.elliptic
+ * @returns {Promise<elliptic>}
+ */
+async function loadEllipticPromise() {
+  const path = config.external_indutny_elliptic_path;
+  const options = config.indutny_elliptic_fetch_options;
+  const ellipticPromise = dl(path, options).catch(() => dl(path, options));
+  const ellipticContents = await ellipticPromise;
+  const mainUrl = URL.createObjectURL(new Blob([ellipticContents], { type: 'text/javascript' }));
+  await loadScript(mainUrl);
+  if(!window.openpgp.elliptic) {
+    throw new Error('elliptic library has not loaded correctly');
+  }
+  URL.revokeObjectURL(mainUrl);
+  return window.openpgp.elliptic;
+}
+
+function loadElliptic() {
+  if(util.detectNode() && config.external_indutny_elliptic) {
+    // eslint-disable-next-line
+    return require('./' + config.external_indutny_elliptic_path);
```

Comment:
I would move this `'./'` to the default value of the config value, as it should always work with it.
```suggestion
    return require(config.external_indutny_elliptic_path);
```

Reply:
fixed